### ISSUE TITLE
[Enhancement] DSPModSave - Notify for missing mods

### ISSUE THREAD

**Issue by starfi5h** [kremnev8/DSP-Mods#77]
In the game Desynced, the game will show the difference of mods used in the save and the current use mods.
![modlist](https://github.com/kremnev8/DSP-Mods/assets/50672801/935da062-f6d7-4773-8f2a-9293f222319f)
So I wonder if DSPModSave should give a notification when there are mods with data that are not currently active. Because  DSPModSave doesn't save mod data of the inactive mods, the data will be clear after the next saving.

I ask this because some players accidentally disabled LossyCompression and lost the shell data. 
Do you think it's worth adding a warning, or that's too annoying for the normal players?

Also, will CommonAPI/DSPModSave/LDBTool still be maintained after the battle update?

**Comment by kremnev8** [comment]
It all will very much depend on how significatly the devs have changed the game. Judjing from how much time has passed, I would be a lot.

**Comment by starfi5h** [comment]
Understand.
Would you accept PR for those mods in the future?
I can try to help fix DSPModSave if it breaks.

**Comment by kremnev8** [comment]
Of course. Unfortunately though my repos for my main mods (IE blueprinttweaks) are not setup for PR’s, but I’m willing to accept help fixing them.
Last, but not least, I will also be looking at updating them myself too in my free time.

**Comment by kremnev8** [comment]
@starfi5h What is the reason you closed the issue? I haven't yet got to looking at this.

**Comment by starfi5h** [comment]
sorry, I thought the info in the log was considered done
